src/main/java/reactor/kafka/receiver/errors/LogAndContinueReceiverExceptionHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
not the typical license header we use, see https://github.com/reactor/reactor-kafka/blob/master/src/main/java/reactor/kafka/receiver/KafkaReceiver.java#L1
and https://github.com/reactor/.github/blob/master/CONTRIBUTING.md#add-apache-license-header-to-all-new-classes